resources/assets/templates/dashboard/DashboardProfile.hbs

```diff
@@ -1,8 +1,19 @@
 <div>
-    <h3>My bookmarks</h3>
+   <h3><a href="#bookmarks">All Bookmarks</a></h3>
+</div>
+<div>
+    <h3><a href="#subscriptions">All subscriptions</a></h3>
+</div>
+
```

Comment:
Please remove blocks with 'All Bookmarks' and 'All Subscriptions' links for now.
We will add them in new layout